firebase-firestore/src/main/java/com/google/firebase/firestore/local/SQLiteIndexManager.java

```diff
@@ -176,6 +176,8 @@ public void addIndexEntries(Document document) {
     if (fieldIndex == null) return null;
 
     Bound lowerBound = target.getLowerBound(fieldIndex);
+    String lowerBoundOp = lowerBound.isBefore() ? ">=" : ">"; // `startAt()` versus `startAfter()`
```

Comment:
Not related to this PR, but what do you think about changing `isBefore` to `isIncluded`?

This way we can do `lower.isIncluded ? ">=" : ">"` and `upper.isIncluded ? "<=" : "<"`. We need to change this everywhere and still keep the API for `startAt` and `startAfter` though.

Reply:
Brian is also confused about this (so am I). I will send a follow-up PR to do a rename.